src/viz/app.py

```diff
@@ -228,56 +225,64 @@ def run_chatbot(store_conversation):  # need store_conversation as input
 
 data = client.list_collections(names_only=False)
 df = pd.DataFrame(data)
+
 pages.append(
     vm.Page(
         title="Data",
-        # layout=GridLayout(grid=[[0, 1, 2], [3, 3, 3], [3, 3, 3]], row_gap="8px", id="data-layout"),
         components=[
             vm.Container(
                 title="Create New Collection",
-                layout=GridLayout(
-                    grid=[[0, 1, 2], [3, 3, 3]], row_gap="8px", id="data-layout"
-                ),
+                layout=GridLayout(grid=[[0, 1, 2], [3, 3, 3]]),
                 components=[
                     UserInput(
                         id="collection_name",
                         title="Collection name",
                         placeholder="default",
                     ),
                     UserInput(id="collection_desc", title="Description"),
-                    UserInput(id="collection_type", title="Type", placeholder="vector"),
+                    vm.Dropdown(
+                        id="collection_type",
+                        title="Type",
+                        options=["vector"],
+                        multi=False,
+                    ),
                     vm.Button(
                         id="add_collection",
-                        text="create",
+                        text="Create",
                         actions=[
                             vm.Action(
                                 function=new_collection(),
                                 inputs=[
-                                    "submit_ingest.n_clicks",
+                                    "add_collection.n_clicks",
                                     "collection_name.value",
                                     "collection_desc.value",
                                     "collection_type.value",
                                 ],
+                                outputs=["data_collections.data"],
                             )
                         ],
                     ),
                 ],
             ),
-            vm.Table(title="Data Collections", figure=dash_data_table(data_frame=df)),
+            vm.Table(
+                title="Data Collections",
+                figure=dash_data_table(id="data_collections", data_frame=df),
+            ),
         ],
     )
 )
 
 sdata = client.list_sessions(short=True)
-print(sdata)
-sdf = pd.DataFrame(sdata)
+# The state column has difficulty with JSONifying and pandas DataFrame, so remove it
```

Comment:
@yaronha This is the reason the table didn't render before - it's just an awkward `state` column that I dropped for now.